src/app/components/NavigationButton/NavigationButton.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+
+type NavigationButtonProps = {
+  children: ReactNode
+}
+export default function NavigationButton({
+  children,
+}: NavigationButtonProps): JSX.Element {
+  return <NavigationButtonContainer>{children}</NavigationButtonContainer>
+}
+const NavigationButtonContainer = styled.div`
```

Comment:
I think that "Container" is completely fine, since you are in the "NavigationButton.tsx", I would know that this container is for the NavigationButton.
```suggestion
const Container = styled.div`
```